### PR TITLE
standalone launch

### PR DESCRIPTION
**Pull request by KeeyanGhoreshi**
This adds a standalone launch option to the DTR app.  The basic gist is that if you attempt to launch the app with an `iss` parameter and no `launch`, it will assume the launch is standalone.  This means that you can skip the CRD portion of the workflow (and skip using the request generator), since the standalone launch does not need it, and wouldn't be able to persist the information returned from the CRD workflow anyway.

To begin a standalone launch, you can go directly to `http://localhost:3005/launch?iss=http://fhir-server-example.com` and the app will attempt to authenticate against the fhir server.  Note that you still need to register the app and get a `client_id`. 

The standalone version of the app provides a patient selection box and loads in-progress questionnaires for the chosen patient.  Then it proceeds like normal.

### CODE REVIEW

File: src/launch.js [hunk 1]
```diff
@@ -108,12 +108,12 @@ function callback(log) {
         serviceUri: log.serviceUri,
         redirectUri: log.redirectUri,
         tokenUri: tokenUri,
-        log: log
+        log: log,
```
Comment (plarocque4) [inline src/launch.js:111]: Extra comma not needed?

**Comment by plarocque4** [comment]
It would be nice if there was some indication of what to do or some type of heading on the standalone launch page. It just ends up having boxes to pick with no information on what it is supposed to do or why.

**Comment by KeeyanGhoreshi** [comment]
addressed above comment in the server-bug-fix, will be implemented when merged into this PR

**Review by plarocque4 — APPROVED** [review]
Looks good

**Pull request merged**